.github/pull_request_template.md

```diff
@@ -13,6 +14,9 @@ Please select the option(s) below that are most relevant:
 - [ ] Bug fix
 - [ ] New feature
 - [ ] Breaking change
+- [ ] GitHub workflow update
+- [ ] Documentation update
+- [ ] Remove legacy code
```

Comment:
"Refactor" might be a better term for this one to cover adding in replacement code?

Reply:
Happy with this change!